cargo-dist/tests/gallery/command.rs

```diff
@@ -21,6 +21,12 @@ impl CommandInfo {
         })
     }
 
+    /// Create a new command, checking that it works by running it with `--version`
```

Comment:
Let's document why we have to do the ext thing, or at least nudge towards checking the axoproject docs for the constant.